ui/src/js/components/labbook/overview/RecentActivity.js

```diff
@@ -125,53 +130,76 @@ export default class RecentActivity extends Component {
   */
   _getDate(edge) {
     const date = new Date(edge.timestamp);
-    return Moment((date)).format('hh:mm a, MMMM Do, YYYY');
+    return Moment((date)).format('hh:mm a');
   }
 
   render() {
     if (this.props.recentActivity) {
       const { owner, labbookName } = store.getState().routes;
       const recentActivity = this.props.recentActivity.slice(0, 3);
+      const edge = this.props.recentActivity[0];
```

Comment:
```suggestion
      const { props, state } = this;
      const edge = props.recentActivity[0];
```
declare props and state like this at the start of your render method. it makes implementations cleaner.